src/vaadin-message.js

```diff
@@ -45,11 +37,27 @@ class MessageElement extends ElementMixin(ThemableMixin(PolymerElement)) {
   static get properties() {
     return {
       /**
-       * Current text value.
+       * Time is rendered as-is to the part='time' slot
        */
-      value: {
+      time: {
         type: String,
         reflectToAttribute: true
+      },
+      /**
+       * A user object that can be used to render avatar and name.
+       * The user object can consist of the folowing properties:
+       * ```js
+       * user: {
+       *   name: string,
+       *   abbr: string,
+       *   img: string,
+       *   colorIndex: number
+       * }
+       * ```
+       */
+      user: {
+        type: Object,
+        reflectToAttribute: true
```

Comment:
Generally it is not recommended to reflect object properties to attributes. Any reason to do it?

Reply:
No specific use. I've heard discussions about other components that it is not nice when the attributes does not reflect what the component is actually showing, so I took it as an practice to reflect back. But I can stop doing that and remove this row.

Reply:
Done.

Reply:
We do not use `reflectToAttribute` for any object / array property, so please consider removing it here.

Reply:
I already did before the "Done." https://github.com/vaadin/vaadin-messages/pull/6/commits/0501ed51bd9935fc7b40c7903c09dd7cf7cea2e3#diff-d54b90d9b433a0b5cd2304cb2f8c1f03ed3fdb917f03c4b45a82ecd3d5d2d3a8L61